server/tests/create-announcement-test.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable consistent-return */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+import announcements from '../data/announcements';
+
+const { should } = chai;
+
+should();
+chai.use(chaiHttp);
+
+describe('Create a new announcement', () => {
+  describe('POST /announcement', () => {
+
+    const defaultUserCredential = {
+      token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZmlyc3RfbmFtZSI6IlNhbSIsImxhc3RfbmFtZSI6IlNtaXRoIiwiZW1haWwiOiJzYW1zbWl0aEBnbWFpbC5jb20iLCJwaG9uZV9udW1iZXIiOiIwNzg0NDQ2MzUyIiwiYWRkcmVzcyI6IktpZ2FsaSIsInBhc3N3b3JkIjoiJDJhJDEwJE5EcE4uVWl5ODNDTk9iMWdmbkhNNmVPUUNDbjA2a21EVHgwOHhnQTNoRjdJbTBZQURDQ20uIiwiaXNfYWRtaW4iOmZhbHNlLCJpc19ibGFjbGlzdGVkIjpmYWxzZSwiaWF0IjoxNTgwMTU3OTc0fQ.B-MZI4D1VVzI8bR7ppR3IRzLE99cz3GTo313W4d42I8'
```

Comment:
how are you getting this token?? I would suggest you create a function that will take the token data, then return a generated token. you can use the same function into your controller and test as well